pkg/optimizely/cache.go

```diff
@@ -38,12 +38,12 @@ type OptlyCache struct {
 	optlyMap        cmap.ConcurrentMap
 	ctx             context.Context
 	wg              sync.WaitGroup
-	conf            config.OptlyConfig
+	conf            config.ProcessorConfig
```

Comment:
Should we rename this field to `processorConf`?

Reply:
no objections